features/lorawan/lorastack/phy/LoRaPHYEU868.h

```diff
@@ -42,317 +42,42 @@
 
```

Comment:
Remove #include "netsocket/LoRaRadio.h" from all region specific header files and also from sources where radio is not referenced.

Reply:
Good idea. May be the next move should be relocate these files.